rules/falco_rules.yaml

```diff
@@ -928,10 +935,12 @@
   items: [sources.list]
 
 - list: repository_directories
-  items: [/etc/apt/sources.list.d, /etc/yum.repos.d]
+  items: [/etc/apt/sources.list.d, /etc/yum.repos.d, /etc/apt]
 
 - macro: access_repositories
-  condition: (fd.filename in (repository_files) or fd.directory in (repository_directories))
+  condition: (fd.directory in (repository_directories) or
```

Comment:
`fd.directory in (repository_directories)` is redundant given the changes of using `pmatch`

Reply:
I think both halves are important--the first half allows for any files directly below repository_directories. The second looks for any of the files in repository_files, but allows them to be multiple levels below repository_directories.

Maybe this is overkill and we can drop the first half, but they are trying to do different things.

Reply:
that's fine, we can keep both.